src/app/page/wallet-show-page/wallet-show-page.component.ts

```diff
@@ -7,4 +7,13 @@ import { Component } from '@angular/core';
 })
 export class WalletShowPageComponent {
 
+  public allWallet: string[];
+  constructor() {
+    this.allWallet = [];
+    const totalWallet: number = 10;
+    this.allWallet.push('+');
```

Comment:
I think the '+' should be at the end of the list. 
Most of the time, user will tap on the existing wallet instead of adding a new wallet, so if we put the '+' button at the back, we will favor most of the users.